_schema.graphql

```diff
@@ -4573,6 +4573,14 @@ type Metaphysics {
   environment: String!
 }
 
+input MoneyInput {
+  # amount
+  amount: Int!
```

Comment:
I don't truly understand what's going on here (with the various references to `Int!`, `number`, and `GraphQLInt`), but would expect this to be a float.

Reply:
oh this is the autogenerated file, but good point it should be float! fixing it in the input definition